### PR TITLE
Adding Roger Chen profile.

### PR DESCRIPTION
**Pull request by rchen27**

### CODE REVIEW

File: people/roger-chen.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Roger Chen
+-------------
+
+![](photos/roger.jpg)
```
Comment (dblock) [inline people/roger-chen.md]: Please rename `roger.jpg` to `roger-chen.jpg`, there're other Roger's I am sure ;)

**Comment by rchen27** [comment]
I renamed the file as per your suggestion. Thanks!

**Comment by dblock** [comment]
Can you please _rebase_ against upstream, and _squash_ your commits. This might be helpful: http://code.dblock.org/2015/08/31/getting-out-of-your-first-git-mess.html

**Comment by rchen27** [comment]
Rebased and squashed commits

**Comment by dblock** [comment]
Very good, thanks!

**Pull request merged**